src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ITodo } from '../../models/ITodo';
+import s from "./TodoItem.module.scss";
+
+const TodoItem: React.FC<{ todo: ITodo }> = (props) => {
+
+  return (
+    <li className={s.todoItem}>
+      <input type="checkbox" />
+      <div className={s.todoItem_message}>
+        <p>{props.todo.message}</p>
```

Comment:
use destructuring to not write "props" all the time

Reply:
Understand, remade this moment for destructuring and then will do the same way.